### PR TITLE
update keycloak guide for 17+

### PR DESCRIPTION
**Pull request by tmnd1991**
Keycloak guide is not up to date and with latest version (18) the path for authentication has changed, making the guide incorrect.

### CODE REVIEW

**Comment by tmnd1991** [comment]
@snazy tagging for visibility

File: site/docs/guides/keycloak.md [hunk 1]
```diff
@@ -61,9 +64,11 @@ docker run -p 19120:19120 \
   -e QUARKUS_OIDC_AUTH_SERVER_URL=http://localhost:8080/auth/realms/master \
   -e QUARKUS_OIDC_CLIENT_ID=projectnessie \
   -e NESSIE_SERVER_AUTHENTICATION_ENABLED=true \
-  --network host projectnessie/nessie-unstable:0.9.3-SNAPSHOT
+  --network host projectnessie/nessie:0.30.0
```
Comment (dimas-b) [inline site/docs/guides/keycloak.md:67]: We have the `latest` tag in docker hub now. Why not use it here?

File: .idea/codeStyles/Project.xml [hunk 2]
```diff
@@ -3,520 +3,15 @@
     <option name="OTHER_INDENT_OPTIONS">
       <value>
         <option name="INDENT_SIZE" value="2" />
-        <option name="CONTINUATION_INDENT_SIZE" value="4" />
         <option name="TAB_SIZE" value="2" />
+        <option name="SMART_TABS" value="true" />
       </value>
     </option>
-    <option name="RIGHT_MARGIN" value="100" />
```
Comment (dimas-b) [inline .idea/codeStyles/Project.xml:10]: These changes seem unrelated to the docs fix. Would you mind removing them from the PR (or perhaps open a new PR just for IDEA project changes)?

**Review by dimas-b — CHANGES_REQUESTED** [review]
Doc changes LGTM, but I guess it is preferable to exclude IDEA project changes from this PR.

**Comment by tmnd1991** [comment]
Resolved your legitimate concerns

**Review by dimas-b — APPROVED** [review]
